src/Engine/Manager/PhysicsManager.cpp

```diff
@@ -78,6 +80,11 @@ void PhysicsManager::Update(float deltaTime) {
         
         // Update rotation.
         entity->rotation += physicsComp->angularVelocity * 360.f * deltaTime;
+
+        physicsComp->GetRigidBody().Position(entity->position);
+        dynamicsWorld->removeRigidBody(physicsComp->GetRigidBody().GetRigidBody());
+        dynamicsWorld->addRigidBody(physicsComp->GetRigidBody().GetRigidBody());
+        physicsComp->GetRigidBody().GetRigidBody()->setGravity(btVector3(0, 0, 0));
     }
 
     dynamicsWorld->stepSimulation(deltaTime, 10);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'MakeRigidBody' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)